runtime/shared_common/j9shr.tdf

```diff
@@ -2935,3 +2935,6 @@ TraceEvent=Trc_SHR_INIT_j9shr_findGCHints_Event_Found Overhead=1 Level=4 Templat
 TraceEvent=Trc_SHR_CM_storeSharedData_OverwriteExisting Overhead=1 Level=4 Template="CM storeSharedData: Existing data in the shared cache has been overwritten (result %p, data->address %p, foundDatalen %zu)."
 TraceEvent=Trc_SHR_CM_updateLocalHintsData_OverwriteHeapSizes Overhead=1 Level=4 Template="CM updateLocalHintsData: Existing hints (heapSize1=%zu, heapSize2=%zu) in the shared cache will be overwritten to (heapSize1=%zu, heapSize2=%zu)."
 TraceEvent=Trc_SHR_CM_updateLocalHintsData_WriteHeapSizes Overhead=1 Level=4 Template="CM updateLocalHintsData: Will write hints (heapSize1=%zu, heapSize2=%zu) to shared cache."
+
+TraceEntry=Trc_SHR_INIT_hookZipOpenState_entry Noenv Overhead=1 Level=1 Template="INIT HOOK ZIP OPEN STATE: Entering hookZipOpenState"
```

Comment:
We don't want to affect performance, I don't think these should be level 1 or 2.

Reply:
Already changed to level 3.